### DISCUSSION TITLE
Script in HEAD not loaded

### DISCUSSION THREAD

**Discussion by ekimkael** [unpoly/unpoly#586] [Q&A] [answered]
I have a basic template where I import tinyMCE. 

```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
  <script src="https://cdn.jsdelivr.net/npm/@tabler/core@1.0.0-beta17/dist/libs/tinymce/tinymce.min.js" defer></script>
</head>
<body>
  
</body>
</html>
```

When I go to the edit page to create or update a resource I have a rich text editor that should be displayed but I continue to have my simple textarea.
<img width="1440" alt="Screenshot 2024-02-09 at 18 53 46" src="https://github.com/unpoly/unpoly/assets/10372405/c4bc4a74-972f-4cd2-a61b-6863b5f97e97">

> N.B: When I reload the application and go to this route, the editor is displayed, but when I quit and come back, it's the textarea.
<img width="1440" alt="Screenshot 2024-02-09 at 18 54 29" src="https://github.com/unpoly/unpoly/assets/10372405/6cb8fe97-a2a5-48a7-8525-1d00ca9cc1a0">

**Comment by alexandremjacques** [comment, marked as answer]
Take a look at: https://github.com/unpoly/unpoly/discussions/584

**Reply by ekimkael** [reply]
Oh My GOD!!! Thanks you so much.
I have been struggling with about a month now. Thank You!!!
